test/unit/service/JobRoleUITest.js

```diff
@@ -0,0 +1,31 @@
+const { Builder, By, Key, until } = require('selenium-webdriver');
+
+
+async function JobRolesUITest() {
+
+  const driver = await new Builder().forBrowser('chrome').build();
+
+  try {
+
+    await driver.get('http://localhost:3000/job_roles');
+
+    const textElement = await driver.findElement(By.xpath("//h1[normalize-space()='View Job Roles']"));
```

Comment:
don't use xpath, if you need to add an ID to the view to get the text then please add one